xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/AnnotateMenuUIX.xml

```diff
@@ -20,22 +20,22 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.2" reference="AnnotationCode.AnnotateMenuUIX" locale="">
+<xwikidoc version="1.3" reference="AnnotationCode.AnnotateMenuUIX" locale="">
   <web>AnnotationCode</web>
   <name>AnnotateMenuUIX</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
-  <creator>xwiki:XWiki.Admin</creator>
+  <creator>XWiki.admin</creator>
   <creationDate>1447859230000</creationDate>
   <parent>AnnotationCode.WebHome</parent>
-  <author>xwiki:XWiki.Admin</author>
-  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1447861877000</date>
-  <contentUpdateDate>1447859231000</contentUpdateDate>
+  <author>XWiki.admin</author>
+  <contentAuthor>XWiki.admin</contentAuthor>
+  <date>1514830377000</date>
+  <contentUpdateDate>1514830377000</contentUpdateDate>
   <version>1.1</version>
   <title/>
-  <comment/>
+  <comment>Install extension [org.xwiki.platform:xwiki-platform-annotation-ui/9.11]</comment>
```

Comment:
Are you sure you ran `xar:format`? Comments should not be included in the committed page.